### PR TITLE
Added required MLH Code of Conduct checkbox

### PR DESCRIPTION
**Pull request by sarj21**
Required per MLH partnership, cant move through form without it

### CODE REVIEW

**Review by mecolmg — COMMENTED** [review]
I think I might use this in the confirmation instead. Probably gonna remove resume too and move it to there.

**Comment by sarj21** [comment]
Yeah that might be better flow

File: app/client/views/confirmation/confirmation.html [hunk 1]
```diff
@@ -365,20 +366,18 @@
...
-            <p>
-              By writing my name below, I affirm that I have read and will abide by the code of conduct detailed above.
-            </p>
-            <input type="text"
-                   name="signatureCodeOfConduct"
-                   placeholder="Your Full Legal Name"
-                   ng-model="user.confirmation.signatureCodeOfConduct">
+            <div class="ui checkbox">
+              <input id="agreeCodeOfConduct"
+              type="checkbox" ng-model="user.confirmation.agreeCodeOfConduct">
```
Comment (mecolmg) [inline app/client/views/confirmation/confirmation.html:375]: You should check the User model to make sure that this is stored properly. Also, if you can find all instances of signatureCodeOfConduct and make sure nothing depends on it that'd be great

Comment (mecolmg) [reply app/client/views/confirmation/confirmation.html:375]: Ignore the part about the user model (sorry I'm on mobile and it won't let me edit the comment).

Comment (sarj21) [reply app/client/views/confirmation/confirmation.html:375]: I checked and those are the only 2 instances. I tested it and it seems ok

Comment (mecolmg) [reply app/client/views/confirmation/confirmation.html:375]: 👍

**Pull request merged**